utils/download_gigaspeech.sh

```diff
@@ -11,9 +11,9 @@ with_dict=false
 
 . ./utils/parse_options.sh || exit 1
 
-if [ $# -ne 1 ]; then
-  echo "Usage: $0 <gigaspeech-dataset-dir>"
-  echo " e.g.: $0 /disk1/audio_data/gigaspeech"
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 <gigaspeech-dataset-dir> <host>"
+  echo " e.g.: $0 /disk1/audio_data/gigaspeech tsinghua/speechocean"
```

Comment:
Can you make the second parameter optional so that this will be backward compatible? Otherwise this will break downstream programs.

Reply:
If the user don't specify that, then we randomly select a host; otherwise, use what the user requires.